src/SuiteSparseMatrixCollection.jl

```diff
@@ -74,6 +74,23 @@ end
 
 ssmc_matrices(ssmc::DataFrame, name::AbstractString) = ssmc_matrices(ssmc, "", name)
 
+"""
+    installed_ssmc()
+
+Return a vector of strings `group/name.format` of all installed matrices from the SuiteSparseMatrixCollection.
+"""
+function installed_ssmc()
+  database = Artifacts.select_downloadable_artifacts(ssmc_artifacts, include_lazy = true)
+  installed_artifacts = String[]
+  for artifact_name in keys(database)
+    hash = Base.SHA1(database[artifact_name]["git-tree-sha1"])
+    if artifact_exists(hash)
+      push!(installed_artifacts, artifact_name)
+    end
+  end
+  return installed_artifacts
```

Comment:
It would make sense if the output of this function could be fed directly to `delete_ssmc`.

Reply:
For example, the `format` argument could be positional in `delete_ssmc` and this could return triples `(group, name, format)` so that we could do
```
for matrix in installed_ssmc()
  detete_ssmc(matrix…)
end
```